examples/cosmos/ics23/ics23.qnt

```diff
@@ -208,8 +208,8 @@ module ics23 {
     // actualPrefix = len(inner.prefix) - 33
     length(inner.prefix) >= ChildSize,
     // getPosition(0) returns 0
-    val from = length(inner.prefix) - ChildSize
-    inner.prefix.slice(from, from + ChildSize) == EmptyChild
```

Comment:
This shouldn't be necessary iiuc? `from` seems to only be parsed out of import statements?

Reply:
The ANTLR grammar introduces  a new token called 'from'. Once we have it, `from` should be a reserved keyword.